rq/job.py

```diff
@@ -493,6 +493,16 @@ def save(self, pipeline=None, include_meta=True):
         connection.hmset(key, self.to_dict(include_meta=include_meta))
         self.cleanup(self.ttl, pipeline=connection)
 
+    def update_meta(self):
```

Comment:
Upon further consideration, could you please change the function signature to "save_meta()"? "update_meta()" sounds like it expects an extra argument. Thanks!